go/vt/vtgate/executor.go

```diff
@@ -1135,7 +1137,7 @@ func (e *Executor) StreamExecute(ctx context.Context, method string, safeSession
 		bindVars = make(map[string]*querypb.BindVariable)
 	}
 	query, comments := sqlparser.SplitMarginComments(sql)
-	vcursor, _ := newVCursorImpl(ctx, safeSession, comments, e, logStats, e.vm, e.VSchema(), e.resolver.resolver)
+	vcursor, _ := newVCursorImpl(ctx, safeSession, comments, e, logStats, e.vm, e.VSchema(), e.resolver.resolver, true)
```

Comment:
Pointing this out as this may be easy to miss: I set `ignoreMaxMemoryRows` to true here as it is being called within `StreamExecute` and I am working under the assumption we do not want to impose row count limits for this method.